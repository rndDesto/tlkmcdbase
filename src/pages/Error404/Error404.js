import React from 'react';
import { Link } from 'react-router-dom';
import { ROUTES } from '../../configs';

export default function Error404() {
  return (
    <section>
      <h2>Page not found</h2>
      <p>
        <Link to={ROUTES.DASHBOARD()}>Back to home</Link>
      </p>
    </section>
  );
}
