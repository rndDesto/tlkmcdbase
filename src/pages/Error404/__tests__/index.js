import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import Error404 from '../Error404';

jest.mock('react-router-dom', () => ({
  Link: function link(props) { return <a {...props} />; },
}));

describe('src/pages/Error404', () => {
  test('render', () => {
    const shallow = new ShallowRenderer();
    const tree = shallow.render(<Error404 />);
    expect(tree).toMatchSnapshot();
  });
});
