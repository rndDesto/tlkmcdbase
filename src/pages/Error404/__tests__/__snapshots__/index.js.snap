// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`src/pages/Error404 render 1`] = `
<section>
  <h2>
    Page not found
  </h2>
  <p>
    <link
      to="/"
    >
      Back to home
    </link>
  </p>
</section>
`;
